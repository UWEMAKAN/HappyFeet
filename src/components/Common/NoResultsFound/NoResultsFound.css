.NoResultsFound {
  color: #cdcdcd;
  font-size: 4rem;
  text-transform: uppercase;
  text-align: center;
  font-weight: 700;
  margin: 15rem 0;
}
