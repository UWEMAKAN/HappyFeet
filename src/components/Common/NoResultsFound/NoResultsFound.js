import React from 'react';
import './NoResultsFound.css';

const NoResultsFound = () => <div className="NoResultsFound">No Results Found</div>;

export default NoResultsFound;
