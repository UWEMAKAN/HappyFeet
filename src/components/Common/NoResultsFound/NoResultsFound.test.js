import React from 'react';
import { shallow } from 'enzyme';
import NoResultsFound from './NoResultsFound';

const renderNoResultsFound = () => shallow(<NoResultsFound />);

test('should render NoResultsFound', () => {
  const wrapper = renderNoResultsFound();
  expect.assertions(2);
  expect(wrapper.text()).toEqual('No Results Found');
  expect(wrapper).toMatchSnapshot();
});
