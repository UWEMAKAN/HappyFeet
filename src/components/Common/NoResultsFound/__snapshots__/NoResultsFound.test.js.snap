// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NoResultsFound 1`] = `ShallowWrapper {}`;
